assets/css/site.css

```diff
@@ -54,6 +58,94 @@ h1 {
 .container .share-bottom {
   margin: 15px 0px 15px 0px;
 }
+
+.container .button {
```

Comment:
I think that `.button` can be confusing. Let's use a more explicitly name, something like `.img-pop-up-button`.